discord/user.py

```diff
@@ -439,5 +470,5 @@ async def create_dm(self):
             return found
 
         state = self._state
-        data = await state.http.start_private_message(self.id)
-        return state.add_dm_channel(data)
+        data: DMChannelPayload = await state.http.start_private_message(self.id)
+        return state.add_dm_channel(data)
```

Comment:
This needs a newline at the end.
```suggestion
        return state.add_dm_channel(data)

```